### ISSUE TITLE
More than just the image is clickable in voting screen

### ISSUE THREAD

**Issue by pachonk** [Mingosio/celebrity-picture-rater#55]
![image](https://user-images.githubusercontent.com/6721355/35073825-62f667f4-fbb0-11e7-9f72-46cb4fbabe39.png)
dashed lines are a border around the img tag.

**Comment by pachonk** [comment]
Changing the height to a max height fixes the issue but places the images at the bottom of the div

**Comment by brandondees** [comment]
you want only the visible image to be clickable, not the whole element? i think it should be straightforward to flex center position the a > img inside a container element and set the fixed size on that container instead

**Comment by grepsedawk** [comment]
Yeah either a good way to show to the user "this whole thing is clickable" _or_ have only the img be clickable.
